GameOfLife/Logic/Game.cs

```diff
@@ -0,0 +1,108 @@
+﻿namespace GameOfLife
+{
+    public class Game
+    {
+        private GameFieldData? gameFieldData;
+        private Renderer renderer = new Renderer();
+
+        /// <summary>
+        /// Main menu of game, which, high level game request logic for proper 
+        /// execution of game of life
+        /// </summary>
+        public void Run()
+        {
+            while (true)
+            {
+                int choice = UserComunicator.GetInputValueRanged(Repository.MainMenuMessage, 1, 3);
+
+                switch (choice)
+                {
+                    case 1:
+                        ExecuteNewGame();
+                        break;
+                    case 2:
+                        AdvanceExistingGame();
+                        break;
+                    case 3:
+                        UserComunicator.PrintWarningMessage(Repository.QuitGameMessage, 0);
+                        Environment.Exit(0);
+                        break;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Creates new game. Sets every time iteration = 1
+        /// so that, when:
+        /// 1.starting new game;
+        /// 2.continuing game
+        /// 3.starting new game.
+        /// Iteration count would not continue to increase but also resets to 1
+        /// Calls constructor for user input and gamearray initialization
+        /// </summary>
+        public void ExecuteNewGame()
+        {
+            GameStateChecker.iterationCount = 1;
+            gameFieldData = new GameFieldData();
+            AdvanceExistingGame();
+        }
+
+        /// <summary>
+        /// Since rendering array increases iteration counter,
```

Comment:
Why does the rendering of an array move the iteration counter forward? It should be just rendering the array and not updating game data

Reply:
Renders only existing data, does not update iterations,